### ISSUE TITLE
Calling inqlinetype before setting lintype causes a crash

### ISSUE THREAD

**Issue by kojix2** [sciapp/python-gr#11]
Hello. Thank you for the great project.
This is a minor issue.

```python
import gr
gr.setlinetype(3)
print(gr.inqlinetype())
```
3

```python
import gr
gr.inqlinetype()
```

```python
import gr
gr.inqlinewidth()
```
Segmentation fault (core dumped)

* Ubuntu 19.04
* GR version 0.41.2

**Comment by FlorianRhiem** [comment]
GR and GKS need to be initialized before being used and many GR functions check whether this has been done yet and do it if necessary. Apparently, some functions expect the user to have used GR before, including some inquire functions. As a result these functions may crash if they are called straight away.

I've prepared a fix for this, adding new autoinit checks to several functions. Until then, a simple workaround would be to call another GR function first which does perform the initialization, e.g. `gr.clearws()` or `gr.inqdspsize()`.

**Comment by kojix2** [comment]
I see. Thank you.

**Comment by FlorianRhiem** [comment]
The issue has been fixed in GR runtime commit [fe68d18331721809200b0e7fe5d1af814be27510](https://github.com/sciapp/gr/commit/fe68d18331721809200b0e7fe5d1af814be27510) and the fix was release as part of GR runtime version [0.41.3](https://github.com/sciapp/gr/releases/tag/v0.41.3).
